packages/grid/x-data-grid-premium/src/components/GridColumnMenuAggregationItem.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { GridColumnMenuItemProps, useGridSelector } from '@mui/x-data-grid-pro';
+import MenuItem from '@mui/material/MenuItem';
+import Stack from '@mui/material/Stack';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import { unstable_useId as useId } from '@mui/utils';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import { styled } from '@mui/material/styles';
+import { useGridApiContext } from '../hooks/utils/useGridApiContext';
+import { useGridRootProps } from '../hooks/utils/useGridRootProps';
+import {
+  canColumnHaveAggregationFunction,
+  getAggregationFunctionLabel,
+  getAvailableAggregationFunctions,
+} from '../hooks/features/aggregation/gridAggregationUtils';
+import { gridAggregationModelSelector } from '../hooks/features/aggregation/gridAggregationSelectors';
+import { GridAggregationModel } from '../hooks/features/aggregation/gridAggregationInterfaces';
+
+const StyledStack = styled(Stack)(({ theme }) => ({
+  padding: theme.spacing(1, 1.5, 1, 1.5),
+}));
+
+const GridColumnMenuAggregationItemRoot: React.FC<GridColumnMenuItemProps> = (props) => {
+  const { column } = props;
+  const apiRef = useGridApiContext();
+  const rootProps = useGridRootProps();
+  const id = useId();
+  const aggregationModel = useGridSelector(apiRef, gridAggregationModelSelector);
+
+  const availableAggregationFunctions = React.useMemo(
+    () =>
+      getAvailableAggregationFunctions({
+        aggregationFunctions: rootProps.aggregationFunctions,
+        column,
+      }),
+    [column, rootProps.aggregationFunctions],
+  );
+
+  const selectedAggregationRule = React.useMemo(() => {
+    if (!column || !aggregationModel[column.field]) {
+      return '';
+    }
+
+    const aggregationFunctionName = aggregationModel[column.field];
+    if (
+      canColumnHaveAggregationFunction({
+        column,
+        aggregationFunctionName,
+        aggregationFunction: rootProps.aggregationFunctions[aggregationFunctionName],
+      })
+    ) {
+      return aggregationFunctionName;
+    }
+
+    return '';
+  }, [rootProps.aggregationFunctions, aggregationModel, column]);
+
+  const handleAggregationItemChange = (event: SelectChangeEvent<string | undefined>) => {
+    const newAggregationItem = event.target?.value || undefined;
+    const currentModel = gridAggregationModelSelector(apiRef);
+    const { [column.field]: columnItem, ...otherColumnItems } = currentModel;
+    const newModel: GridAggregationModel =
+      newAggregationItem == null
+        ? otherColumnItems
+        : { ...otherColumnItems, [column?.field]: newAggregationItem };
+
+    apiRef.current.setAggregationModel(newModel);
+  };
+
+  const label = apiRef.current.getLocaleText('aggregationMenuItemHeader');
+
+  return (
+    <FormControl variant="standard" size="small" fullWidth>
```

Comment:
Maybe outlined variant for aggregation select would look better? cc @gerdadesign 
```suggestion
    <FormControl variant="outlined" size="small" fullWidth>
```

<img width="276" alt="Screenshot 2022-11-18 at 19 51 20" src="https://user-images.githubusercontent.com/13808724/202780757-60a4b4d3-4332-468c-b972-6e0b8abed8ca.png">

Reply:
Yes, looks better to me too 👍